mod/01_core/set/all/fetch.rb

```diff
@@ -55,7 +55,9 @@ def fetch mark, opts={}
       if opts[:new]
         return card.renew(opts) if !clean_cache_opts? opts
       else
-        return unless !opts[:skip_virtual] && card.virtual?
+        card.include_set_modules unless opts[:skip_modules]  # need to load modules here to call the right virtual? method 
+        opts[:skip_modules] = true                           # don't load twice
+        return if opts[:skip_virtual] || !card.virtual?
```

Comment:
Let me see if I get this.  Before, the card would not be virtual? here when it the fact that it was virtual? was defined in code only.

But wait, wouldn't the card 'Home+*followers' already be virtual? if there is a rule *followers+*right+*structure ?  I can see that you might put that completely in code, but would it be a solution to create this structure card anyway?  Presumably you are defining :raw and :core views for the set in code, but why not create the support to know it is virtual without the model override for virtual?